src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/patternvariablename/Example3.java

```diff
@@ -0,0 +1,25 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="PatternVariableName">
+        <property name="format" value="^[a-z][_a-zA-Z0-9]{2,}$"/>
+    </module>
+  </module>
+</module>
+*/
+
+//non-compiled with javac: Compilable with Java17
+package com.puppycrawl.tools.checkstyle.checks.naming.patternvariablename;
+
+// xdoc section -- start
+class Example3 {
+  Example3(Object o1){
+    if (o1 instanceof String STRING) { // violation
+    }
+    if (o1 instanceof Integer num) {
+    }
+    if (o1 instanceof Integer num_1) {
```

Comment:
please make some change in this example to make it different from Example2 by violations.

Reply:
Added new set of violation and made all the examples same by java code.
Please review